itou/templates/search/includes/siaes_search_results.html

```diff
@@ -0,0 +1,33 @@
+<div id="job-search-results" class="col-12 col-md-8">
+    {% for item in results_page %}
+        {% if request.resolver_match.view_name == "search:employers_results" %}
+            {% include "companies/includes/_card_siae.html" with siae=item %}
+        {% else %}
+            {% include "companies/includes/_card_jobdescription.html" with job_description=item %}
+        {% endif %}
+    {% empty %}
+        <div class="c-box c-box--results mb-3 mb-md-4">
+            <div class="c-box--results__body">
+                <p class="mb-0">Aucun résultat avec les filtres actuels.</p>
+            </div>
+        </div>
+    {% endfor %}
+    {% include "includes/pagination.html" with page=results_page boost=True boost_target="#job-search-results" boost_indicator="#job-search-results" %}
+</div>
+{% if request.htmx %}
+    <title id="page-title" hx-swap-oob="true">
```

Comment:
```suggestion
    <title hx-swap-oob="outerHTML title">
```
pourrait a priori également fonctionner (mais demanderait de rajouter le support dans `tests/utils/test.py`

Reply:
C’est bien mieux si ça fonctionne :+1:

Reply:
Bien vu l'update du <title> 👍

Reply:
Et ça fonctionne très bien, merci :)